pkg/sources/github/github.go

```diff
@@ -483,16 +483,14 @@ func (s *Source) getReposByOrg(ctx context.Context, apiClient *github.Client, or
 	var numRepos, numForks int
 	for {
 		someRepos, res, err := apiClient.Repositories.ListByOrg(ctx, org, opts)
-		if err == nil {
-			defer res.Body.Close()
-		}
+		res.Body.Close()
 		if handled := handleRateLimit(err, res); handled {
 			continue
 		}
 		if err != nil {
 			return nil, fmt.Errorf("could not list repos for org %s: %w", org, err)
 		}
-		log.Debugf("listed repos page %d/%d", opts.Page, res.LastPage)
+		s.log.Debugf("Listed repos for org %s page %d/%d", org, opts.Page, res.LastPage)
```

Comment:
question: Not sure if the check for `err != nil` means res != nil, because if err is not nil but `res` is still nil `res.LastPage` will panic.

NOTE: this applies to the debugs below as well.